docs/developer/CONTRIBUTING_HERE.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to jupyter/accessibility directly
+
+This contribution guide is for people who want to add knowledge to the jupyter/accessibility repository. If you are instead looking to contribute to accessibility in the Jupyter ecosystem, see [here](../../CONTRIBUTING.md).
```

Comment:
I am assuming your editor is wrapping text at a certain lenght - can you split this at 120 char? it makes it so much easier to review PRs 🙏🏼